observability-lib/api/notification-policy_test.go

```diff
@@ -7,6 +7,10 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func Pointer[T any](d T) *T {
```

Comment:
we tend to replicate something like this across many of our testing packages. 
https://smartcontract-it.atlassian.net/browse/BCF-3448. No need to fix in this PR